kotlin-guiced.gradle.kts

```diff
@@ -51,14 +63,48 @@ subprojects {
         }
     }
 
-    repositories {
-        mavenCentral()
+    dependencies {
+        "compile"(kotlin(module = "stdlib", version = property("kotlin.version") as String))
+        "compile"(kotlin(module = "reflect", version = property("kotlin.version") as String))
+
+        "testCompile"(junitJupiter("junit-jupiter-api"))
+        "testCompile"(junitJupiter("junit-jupiter-params"))
+        "testRuntime"(junitJupiter("junit-jupiter-engine"))
+        "testRuntime"(create(group = "org.junit.platform", name = "junit-platform-launcher", version = "1.1.0-M1"))
     }
 
-    dependencies {
-        "compile"(kotlin("stdlib"))
-        "compile"(kotlin("reflect"))
-        "testCompile"(create(group = "io.kotlintest", name = "kotlintest", version = "2.0.2"))
+    junitPlatform {
+        details = Details.VERBOSE
+
+        filters {
+            includeClassNamePatterns(".*Test", ".*Tests", ".*Spec")
+        }
+    }
+
+    // Below, configure jacoco code coverage on all Junit 5 tests.
```

Comment:
Okay, all of these Jacoco changes probably should have been in a different PR. But whatever.